docs/core/integration-quality-scale/rules/iqs024.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "IQS024 - Above average test coverage for all integration module"
```

Comment:
I suggest we change the rule to say above 95% coverage. Above average worked before when the project average was high. With the current average of around 80 % I don't think it's a good reference.

```suggestion
title: "IQS024 - Above 95 % test coverage for all integration modules"
```